packages/flutter/test/material/text_field_test.dart

```diff
@@ -5674,4 +5674,71 @@ void main() {
     );
     expect(topLeft.dx, equals(383)); // Should be same as equivalent in 'Caret center position'
   });
+
+  testWidgets('shows selection carets', (WidgetTester tester) async {
+    const String testText = 'lorem ipsum';
+    final TextEditingController controller = TextEditingController(text: testText);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: TextField(
+            controller: controller,
+          ),
+        ),
+      ),
+    );
+
+    final EditableTextState state =
+        tester.state<EditableTextState>(find.byType(EditableText));
```

Comment:
It's OK to just put this variable and its value on one line; otherwise 2-space indent (here and below).